infrastructure/main.tf

```diff
@@ -70,6 +70,18 @@ data "vault_generic_secret" "oauth_client_secret" {
   path = "secret/${var.vault_section}/ccidam/idam-api/oauth2/client-secrets/cmc-citizen"
 }
 
+module "scheduler" {
+  source = "git@github.com:hmcts/moj-module-postgres?ref=cnp-449-tactical"
```

Comment:
Please pass in a version parameter and set it to 10.3 it just got released a couple of days ago better to do now rather than later